source/examples/types/MceStartStopIO/definition.yaml

```diff
@@ -2,6 +2,19 @@ company: Yaskawa Europe GmbH
 author: Rioual
 comment: IO data for the MceStartStop function
 changelog:
+  - version: 0.3.0
```

Comment:
To avoid mismatch between FB and IOs data structure version, should we match the FB version number ?
If yes, we should jump to version `1.0.0`.

Otherwise we jump from `0.2.0` to `0.3.0`.